crates/forge/src/cmd/remove.rs

```diff
@@ -31,7 +31,21 @@ impl_figment_convert_basic!(RemoveArgs);
 impl RemoveArgs {
     pub fn run(self) -> Result<()> {
         let config = self.load_config()?;
-        let (root, paths, _) = super::update::dependencies_paths(&self.dependencies, &config)?;
+        let (root, mut paths, _) = super::update::dependencies_paths(&self.dependencies, &config)?;
```

Comment:
shouldn't this be done inside of the function?

Reply:
updated in https://github.com/foundry-rs/foundry/pull/11826/commits/613462a810729d565d8fdbd4c0f11ee2b1116550